### DISCUSSION TITLE
Why did some of the 🛶 in Venice turn 💚 a few weeks ago?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#917] [Galaxy Trivia] [answered]
**Bonus:** Ask the AI to create an art drawing of it with a gondolier standing on a paddleboard.

**Comment by peterbaccaro** [comment, marked as answer]
Because of the presence of **fluorescein** in the water, an environmentally harmless dye that can be used to locate blood at crime scenes and diagnose eye injuries. It can also help determine whether sewer systems are working properly.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/d7a38bd6-a199-413a-ae9e-354f2fb6b36f)
